chef_master/source/install_windows.rst

```diff
@@ -20,8 +20,6 @@ The Chef Infra Client can be installed on machines running Microsoft Windows in
 * By downloading Chef Infra Client to the target node, and then running the Microsoft Installer Package (MSI) locally
 * By using an existing process already in place for managing Microsoft Windows machines, such as System Center
 
-To run Chef Infra Client at periodic intervals (so that it can check in with Chef Infra Server automatically), configure Chef Infra Client to run as a service or as a scheduled task. (The Chef Infra Client can be configured to run as a service during the setup process.)
```

Comment:
We still want the mention of the scheduled task. Just the service part needs to DIAF

Reply:
Setting up as a scheduled task is covered in the doc.